lib/plugins/readme.md

```diff
@@ -80,3 +80,7 @@ Note that `publish` and `unpublish` are used by certain plugins, but they will b
   - hook triggered on page unscheduling
   - _note:_ when `unschedulePage` is triggered, `save` is also triggered
   - given one param with uri, data, and user: `{ uri: scheduledItemUri, data: { at: timestamp, publish: pageUri }, user: req.user }`
+
+### Layout-specific Methods
+
+Layouts only have one hook, called `publishLayout`. It is called with the same arguments as `publishPage`, namely `{ uri, data, user }`.
```

Comment:
I'd keep out nix "It is called with the same arguments as `publishPage`" in case you forget to remove that if the arguments for publishPage change and this doesn't.  Just including the args should suffice!

Reply:
I really don't want those to diverge though! This comment establishes that intent